src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
 // write code here
+const tbody = document.querySelector('tbody');
+const rows = Array.from(tbody.rows);
+
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColBtn = document.querySelector('.append-column');
+const removeColBtn = document.querySelector('.remove-column');
+
+appendRowBtn.addEventListener('click', () => {
+  let rowCount = tbody.getElementsByTagName('tr').length;
+
+  rowCount++;
+
+  const row = document.createElement('tr');
+
+  for (let i = 0; i < rows[0].cells.length; i++) {
+    row.appendChild(document.createElement('td'));
+  }
+  tbody.appendChild(row);
+  rows.push(row);
+
+  if (rowCount === 10) {
+    appendRowBtn.setAttribute('disabled', 'disabled');
+  }
+
+  if (rowCount > 2) {
+    removeRowBtn.removeAttribute('disabled');
+  }
+});
+
+removeRowBtn.addEventListener('click', () => {
+  let rowCount = tbody.getElementsByTagName('tr').length;
+
+  rowCount--;
+  removeRowBtn.removeAttribute('disabled');
```

Comment:
This line is unnecessary as the `removeRowBtn` should only be enabled if the row count is greater than 2, which is already handled in the subsequent condition.